test/functional/index.js

```diff
@@ -44,6 +44,18 @@ describe('Autocomplete', () => {
         expect(scratch.innerHTML).to.contain('name="input-autocomplete"')
       })
 
+      it('renders an input with a required attribute', () => {
+        render(<Autocomplete required />, scratch)
+
+        expect(scratch.innerHTML).to.contain('required')
+      })
+
+      it('renders an input without a required attribute', () => {
+        render(<Autocomplete required={false} />, scratch)
```

Comment:
I think that would be testing the vdom library and not our own code, so I don't think it's necessary.